### ISSUE TITLE
appveyor windows compile

### ISSUE THREAD

**Issue by gambhiro** [pyros2097/rust-embed#66]
I noticed something when trying to use the git repo directly in Cargo.toml.

```
rust-embed = { git = "https://github.com/pyros2097/rust-embed" }
```

Compiling with Appveyor, the build errored out with this message:

```
Compiling rust-embed-impl v4.4.0 (https://github.com/pyros2097/rust-embed#de8e97fe)
error: expected item, found `..`
 --> C:\Users\appveyor\.cargo\git\checkouts\rust-embed-88640eaf411660a2\de8e97f\impl\src\utils.rs:1:1
  |
1 | ../../src/utils.rs
  | ^^ expected item
error: aborting due to previous error
error: Could not compile `rust-embed-impl`.
```

The latest released version (v4.4.0) was building successfully.

**Comment by Mcat12** [comment]
That file is a symlink, which it seems Windows may not support. @pyros2097 why was this done?

**Comment by pyros2097** [comment]
Ohh yeah this was done by @vemoo when separating the rust-embed package to 2 packages (rust-embed and rust-embed-impl). This was done because of some limitation of proc-macros.
Here is the PR,
https://github.com/pyros2097/rust-embed/pull/45

Maybe we can do something to solve the symlink.

**Comment by Mcat12** [comment]
There's no reason to include the `utils.rs` file in the main package, it isn't used at all and is hidden from the docs. I will move it to the `impl` package.

**Comment by Mcat12** [comment]
Because the code is used in the macro output, it has to be accessible outside the crate. Since the `impl` crate can't expose anything other than the macro, the main crate has to expose the functionality. However, the `impl` crate also needs to use the functionality in certain cases (release mode), so it has to be accessible there. We can't import the main crate into the `impl` crate, so we need to make a new "common" crate.

**Comment by pyros2097** [comment]
Yeah thats what I figured maybe a rust-embed-utils crate to be used by both the crates I guess.

**Comment by AletiCodes** [comment]
I was looking forward to using this package for Windows-only app but ran into the above issue... are there any plans to fix this?

**Comment by pyros2097** [comment]
Well we can fix it. we just need to create a new utils package for the utils.rs file and add it to the other 2 crates. Or the easy way is copying that file into both the packages. Will be interested in a PR. 😃

**Comment by AletiCodes** [comment]
> 
> 
> Well we can fix it. we just need to create a new utils package for the utils.rs file and add it to the other 2 crates. Or the easy way is copying that file into both the packages. Will you be interested in a PR. 😃
> 
> Also you can use a fixed version like v4.5.0 and it would work

I went ahead and made a PR for this. (#71)

**Comment by pyros2097** [comment]
release v5.0.0 https://github.com/pyros2097/rust-embed/pull/72
